packages/starlight-view-modes/translations.ts

```diff
@@ -11,4 +11,10 @@ export const Translations = {
     "starlightViewModes.defaultMode.title": "normaler Modus",
     "starlightViewModes.zenMode.title": "Zen-Modus",
   },
+  fr: {
+    "starlightViewModes.switchesHeading": "Modes de visualisation",
+    "starlightViewModes.switchTo": "Basculer vers {{ text }}",
+    "starlightViewModes.defaultMode.title": "Mode normal",
```

Comment:
I'm no French native, but I think translating it to French makes sense, so I'll take that! Thank you sooo much!

```suggestion
    "starlightViewModes.defaultMode.title": "Mode par défaut",
```

Reply:
Yeah both works, but for me using `par défaut` feels more natural than `normal` in this context. So I think this update makes sense although I don't have a usage background to tell which is more common.